src/modules/m_nick.c

```diff
@@ -298,13 +298,13 @@ void nick_identify (Nick *nptr, User *uptr __unused, char *all)
     if (!user) {
         NoticeToUser(nptr,"You are not registered");
         return;
-    }   
-    
-    if (nptr->svid != NULL) {
+    }
+    if (nptr->svid[0] != '\0')
```

Comment:
See comment on the previous commit.

Reply:
Switched back to user->authed as it makes more sense in this PR (there is no account/nick separation yet)